src/action/base/create_user.rs

```diff
@@ -219,16 +219,11 @@ impl Action for CreateUser {
     }
 
     fn revert_description(&self) -> Vec<ActionDescription> {
-        let Self {
-            name,
-            uid,
-            groupname: _,
-            gid,
-            action_state: _,
-        } = self;
-
         vec![ActionDescription::new(
-            format!("Delete user {name} with UID {uid} with group {gid}"),
+            format!(
+                "Delete user `{}` (UID {}) in group {} (GID {})",
```

Comment:
Since you backquote the username, is it worth it to backquote the group name?

Reply:
You're right!!!!